test/browser/home-page.test.jsx

```diff
@@ -0,0 +1,37 @@
+var should = require('should');
+var sinon = window.sinon;
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var stubContext = require('./stub-context.jsx');
+var HomePage = require('../../pages/home.jsx');
+
+describe("HomePage", function() {
+  var blogSection;
+
+  beforeEach(function() {
+    blogSection = stubContext.render(HomePage.BlogSection);
```

Comment:
Yeah, I think that's actually a really good idea--it follows the general philosophy of unit testing, where you want to test each component in isolation.

I'd just rename the `describe("HomePage"` to `describe("HomePage.BlogSection"`. That way if we ever add anything else to the homepage that needs testing, we can just put it in a separate `describe` call rather than adding it into the existing one, which complicates the testing logic and makes it more confusing to understand what's going on.